BUILD.bazel

```diff
@@ -5,11 +5,14 @@ alias(
     actual = "@nodejs//:yarn",
 )
 
+alias(
```

Comment:
maybe a comment that this is the place ts_library will look by default, allows us to omit explicit tsconfig attributes?

Reply:
👍